src/receiver.rs

```diff
@@ -60,11 +60,11 @@ impl Receiver {
 
     pub fn decrypt<EncryptedFrame>(
         &mut self,
-        encrypted_frame: &EncryptedFrame,
+        encrypted_frame: EncryptedFrame,
         skip: usize,
     ) -> Result<&[u8]>
     where
-        EncryptedFrame: AsRef<[u8]> + ?Sized,
+        EncryptedFrame: AsRef<[u8]>,
```

Comment:
actually the `?Sized` relaxed the constraint. Now the signature requires a Sized type. Which is the opposite of what your commit message sais.

Reply:
You are right, the commit message is wrong.
What I intended with that change is, that one can now pass those arguments by moving. Parameters with an Unsized Type still can be passed by reference with this change.
I will adapt the commit message.
I missed some more places where we have the `?Sized` bound and enforce passing by reference unnecessarly.